api-migration.md

```diff
@@ -15,6 +15,29 @@ The library continues to support legacy plugins. They can be used in conjunction
 with new plugins. Your plugins list may contain any number of new plugins but
 only ever one legacy plugin.
 
+It's important to note that the `cloudant.request` object is a pass through
+stream and isn't always suitable for piping. However, the response object
+returned by the event hook can always be piped. This is because it's not
+guaranteed that any given response will be returned to the client until all the
```

Comment:
`This is because...` - this sentence is awkward.  Is there a way to simply it with the previous sentence? Maybe something like:
`If piping is required, use the response object returned by the event hook since the client will return a complete response once all 'onResponse' hooks have been executed.`

Reply:
Again, it can only be used outside the event hook. Otherwise use the response object returned by the event hook. Will try and clarify.